packages/components/src/DataList/components/DataListBulkActions/DataListBulkActions.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { Children, ReactElement, isValidElement } from "react";
+import { Tooltip } from "@jobber/components/Tooltip";
+import { Button } from "@jobber/components/Button";
+import { DataListBulkActionsProps } from "../../DataList.types";
+import { useDataListContext } from "../../context/DataListContext";
+import { DataListItemActionsOverflow } from "../DataListItemActions/DataListItemActionsOverflow";
+import { getExposedActions } from "../../DataList.utils";
+import { DataListOverflowFade } from "../DataListOverflowFade";
+
+// This component is meant to capture the props of the DataList.BulkActions
+export function DataListBulkActions(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _: DataListBulkActionsProps,
+) {
+  return null;
+}
+
+export function InternalDataListBulkActions() {
+  const { bulkActionsComponent } = useDataListContext();
+  if (!bulkActionsComponent) return null;
+
+  const { children } = bulkActionsComponent.props;
+  const childrenArray =
+    Children.toArray(children).filter<ReactElement>(isValidElement);
+
+  const exposedActions = getExposedActions(childrenArray, 3);
+  childrenArray.splice(0, exposedActions.length);
+
+  return (
+    <DataListOverflowFade>
+      {exposedActions.map(({ props }) => {
+        if (!props.icon) return null;
+
+        return (
+          <Tooltip key={props.label} message={props.label}>
+            <Button
+              icon={props.icon}
+              ariaLabel={props.label}
+              onClick={() => props.onClick?.()}
+              type="secondary"
+              variation="subtle"
+            />
+          </Tooltip>
+        );
+      })}
+
+      <DataListItemActionsOverflow actions={childrenArray} />
```

Comment:
This is _almost_ the exact same code as https://github.com/GetJobber/atlantis/blob/ed7f30d2456b389487281bd0eaeb36ab7101336d/packages/components/src/DataList/components/DataListItemActions/DataListItemActions.tsx#L68-L84

I think we can just create a `DataListActions` component that gives you this code + this code https://github.com/GetJobber/atlantis/blob/ed7f30d2456b389487281bd0eaeb36ab7101336d/packages/components/src/DataList/components/DataListItemActions/DataListItemActions.tsx#L48-L53

and accepts a prop of 
- the list of actions children
- how many to expose (2 or 3)

As for the item, you can grab that from `useDataListLayoutActionsContext` so no need for prop drilling

Could be a follow up tho